RecoLocalCalo/HcalRecAlgos/src/HcalDeterministicFit.cc

```diff
@@ -3,6 +3,10 @@
 #include <climits>
 #include "RecoLocalCalo/HcalRecAlgos/interface/HcalDeterministicFit.h"
 
+const float HcalDeterministicFit::invGpar[3];
+const float HcalDeterministicFit::negthre[2];
+const int HcalDeterministicFit::HcalRegion[2];
+
```

Comment:
why is landauFrac ( of ``` static constexpr float landauFrac[] { initialization } ``` ) from the header file doesn't have to be in .cc?
All of these are constexpr in .h. Should a text object still be defined in the object file/library?